docs/installation.adoc

```diff
@@ -179,7 +179,7 @@ $ dnf install odo
 ==== Tarball installation
 
 ----
-# sh -c 'curl -L https://github.com/openshift/odo/releases/latest/download/odo-linux-amd64.gz | gzip -d > /usr/local/bin/odo; chmod +x /usr/local/bin/odo'
+# sh -c 'curl -L https://github.com/openshift/odo/releases/latest/download/odo-linux-amd64.tar.gz | tar -xvzf odo-linux-amd64.tar.gz -C /usr/local/bin; chmod +x /usr/local/bin/odo'
```

Comment:
@amitkrout this change

Reply:
https://github.com/openshift/odo/pull/2009#discussion_r316532573